### ISSUE TITLE
How to upload and save video file on other FTP Server 

### ISSUE THREAD

**Issue by Kulteam** [MacWarrior/clipbucket-v5#233]
I want save video on second FTP but cannot find any script I need
 I try Clip Bucket but cannot found this option !

**Comment by joo4mart** [comment]
Please can you give us more detailled hints: 
What do you mean in detail with 'save video' and with 'second FTP'?
Have you installed ClipBucket v5 with the install script? At with operating system?

**Comment by joo4mart** [comment]
I still don't understand 'save video' because ClipBucket is an alternative to Youtube and not to store files on other servers. Or do you mean 'upload' instead of 'save'?
Laragon (see: https://laragon.org/download/) still use MySQL 5.7. This is a very old version and I think it's not compatible to ClipBucket 5.5 (beta).

**Comment by Kulteam** [comment]
Yes. I want to stream and upload to other storage hosting like Wasabi,AWS S3..or other FTP server.

**Comment by MacWarrior** [comment]
Hello

Clipbucket is build to work with directories ; if you can find a way to mount your distant storage, then it will work fine
From my side, I use a mounted NFS as storage and symbolic links to replace /upload/files directory

FTP/SFTP, S3 & Wasabi seems to be mountable as directories (Google is your best friend for this step as it's a hosting subject and not directly related to CB)

> I try install ClipBucket on Laragon to test

Never heard ot it, but CB should work fine on MySQL 5.X

Best regards,
MacWarrior.

**Comment by anhthoai** [comment]
> Yes. I want to stream and upload to other storage hosting like Wasabi,AWS S3..or other FTP server.

I have developed and used plugin that can do it. After converting, videos files will be uploaded to Wasabi, AWS S3, Minio, BunnyCDN or any FTP Server

**Comment by Guber01** [comment]
Yes, it would be quite comfortable and great!. So done in many engines, for example DLE. Just load the file, it is processed by ffmpeg. Then he is loaded to the service that we would like to use (Webdav, NFS, FTP and others) and generate a link to this video or photo. In the case of FTP, simply after processing, you load the FTP file wherever we need. FTP folder must be visible through the browser. In the settings, simply indicate two paths - the path to FTP and the path in the web folder of this FTP. It should also be done so ... if the service is not available, so that the file is loaded either on the backup service or on the host drive.
